core/src/main/scala/com/karumi/shot/android/Adb.scala

```diff
@@ -9,13 +9,28 @@ object Adb {
 }
 
 class Adb {
+
+  final val CR_ASCII_DECIMAL = 13
+
   def devices: List[String] = {
-    executeAdbCommandWithResult("devices").split('\n').toList.drop(1).map {
+    listWithoutCarriageReturn(executeAdbCommandWithResult("devices").split('\n').toList.drop(1).map {
```

Comment:
Instead of doing this why don't you filter the lines you don't want?

Reply:
I solve the request, I don't test the changes I will do this later and create the tests for this change, I check and don't have tests for the Adb class